develop/application-devs/debugging-errors.md

```diff
@@ -1,72 +1,105 @@
 ---
 title: Errors and How to Resolve Them
-description: Common errors that are encountered across the runtime, tooling, and block explorers.
+description: This page provides an overview of frequently encountered errors and essential tools, like Polkadot.js Apps, to help you identify and resolve common issues.
 ---
 
-Errors in Substrate-based chains are usually accompanied by descriptive messages. However, to read
-these messages, a tool parsing the blockchain data needs to request _chain metadata_ from a node.
-That metadata explains how to read the messages. One such tool with a built-in parser for chain
-metadata is the [Polkadot-JS Apps UI](https://polkadot.js.org/apps){target=\_blank}.
+# Troubleshooting Common Errors
 
-If this page doesn't answer your question, try searching for your problem at the
-[Polkadot Knowledge Base](https://support.polkadot.network/){target=\_blank} for more information on troubleshooting
-your issue.
+Errors in Substrate-based chains are typically accompanied by descriptive messages. To interpret these messages, tools that parse blockchain data must request _chain metadata_ from a node, which provides the necessary information to decode them. One such tool is the [Polkadot.js Apps UI](https://polkadot.js.org/apps){target=\_blank}, which includes a built-in parser for chain metadata that allows error messages to be displayed in a human-readable format.
```

Comment:
Also, are we sticking to "Polkadot SDK-based" or also using "Substrate-based?"

Reply:
Ah, I'm out of the loop. It's Polkadot SDK-based now. Thanks!